project/MiMa.scala

```diff
@@ -534,7 +534,10 @@ object MiMa extends AutoPlugin {
       FilterAnyProblem("akka.remote.RemoteWatcher$UnwatchRemote"),
       FilterAnyProblem("akka.remote.RemoteWatcher$Rewatch"),
       FilterAnyProblem("akka.remote.RemoteWatcher$RewatchRemote"),
-      FilterAnyProblem("akka.remote.RemoteWatcher$Stats")
+      FilterAnyProblem("akka.remote.RemoteWatcher$Stats"),
+
+      // toString is available on any object, mima is confused due to a generated toString appearing #17722
+      ProblemFilters.exclude[MissingMethodProblem]("akka.japi.Pair.toString")
```

Comment:
@rkuhn It's

```
[info] akka-actor: found 1 potential binary incompatibilities (filtered 232)
[error]  * method toString()java.lang.String in object akka.japi.Pair does not have a correspondent in new version
[error]    filter with: ProblemFilters.exclude[MissingMethodProblem]("akka.japi.Pair.toString")
```

https://github.com/akka/akka/pull/17722#issuecomment-114485665